core/services/vrf/listener_v2.go

```diff
@@ -64,7 +64,7 @@ type listenerV2 struct {
 	job            job.Job
 	q              pg.Q
 	gethks         keystore.Eth
-	reqLogs        *utils.Mailbox
+	reqLogs        *utils.Mailbox[log.Broadcast]
```

Comment:
I believe there's a code block that does a cast in `runLogListener` that can be removed, unless I'm looking at older code:

```golang
lb, ok := i.(log.Broadcast)
if !ok {
	panic(fmt.Sprintf("VRFListenerV2: invariant violated, expected log.Broadcast got %T", i))
}
```